src/web/core/src/main/java/org/geoserver/web/data/resource/BasicResourceConfig.html

```diff
@@ -8,6 +8,10 @@ <h3><wicket:message key="title">Basic Settings</wicket:message></h3>
       <li>
         <fieldset>
           <legend><span><wicket:message key="basicResourceInfo">Basic Resource Info</wicket:message></span></legend>
+          <div>
+            <label><wicket:message key="storeName">Store Name</wicket:message></label>
```

Comment:
As per the [JIRA ticket](https://osgeo-org.atlassian.net/browse/GEOS-9677), what we want to see is the native name of the resource behind the layer, not the store name. For example, for a relational database what should be displayed is the of the table that will be used by the layer, not the relational database store name.

Reply:
I guess we could actually add both the store name and the native name.

Reply:
It would not hurt indeed.

Reply:
I added both labels.